app/controllers/comments_controller.rb

```diff
@@ -33,6 +34,9 @@ def create
       if @comment.save
         format.html { redirect_back fallback_location: root_path, notice: "Comment was successfully created." }
         format.json { render :show, status: :created, location: @comment }
+        format.js do
+          render template: "comments/create"
+        end
```

Comment:
Good. Additionally, you can utilize the shorthand `format.js`. This built-in Ruby method will automatically locate the JavaScript file named 'create'.